config-1_x_0.md

```diff
@@ -142,7 +135,7 @@ rescue:
 
 ### Rescue
 
-Rescue blocks are used to catch exceptions and handle HTTP status codes generated by the particular handler.
+Rescue blocks help to catch exceptions and handle HTTP status codes generated by the particular handler.
```

Comment:
Not sure about using "help" here. It sounds like they optimize something, make it easier to implement. In fact, they are the only way made specifically to catch exceptions.